vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -225,6 +226,9 @@
 	elsif ( $token eq "-use_odin_simulation" ){
 		$odin_run_simulation = 1;
 	}
+	elsif ( $token eq "-custom_odin_config_full_name" ){
```

Comment:
`-odin_xml` shorter and odin only support xml for one thing other than architecture, which are already specified here, so it disambiguates itself.